### PR TITLE
docs: documented visual studio requirements

### PR DESCRIPTION
**Pull request by ghuntley**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Other information

If there's anything missing please edit the PR then merge.

### CODE REVIEW

File: doc/articles/getting-started/requirements.md [hunk 1]
```diff
@@ -9,16 +9,20 @@ The platform requirements for the Uno Platform are as follows:
...
+The minimumn supported version by the Uno Platform is iOS 8 from 2014.
 
 ## Universal Windows Platform (UWP)
 
 When developing for the Universal Windows Platform (UWP) there are no special requirements. The Uno Platform is a Universal Windows Platform (UWP) Bridge that allows UWP based code to run on iOS, Android, and WebAssembly. Uno provides the full API definitions of the UWP Windows 10 October 2018 Update (17763), and the implementation of parts of the UWP API, such as Windows.UI.Xaml, to enable applications to run on these platforms.
 
 ## WebAssembly
 
-- WebAssembly is supported in Chrome, Edge, Firefox and Safari. See the official WebAssembly site for [more details](https://webassembly.org/roadmap/).
+WebAssembly is supported in Chrome, Edge, Firefox and Safari. See the official WebAssembly site for [more details](https://webassembly.org/roadmap/).
```
Comment (jeromelaban) [inline doc/articles/getting-started/requirements.md]:
```suggestion
WebAssembly is supported in Chrome, Edge, Edge Dev, Firefox and Safari. See the official WebAssembly site for [more details](https://webassembly.org/roadmap/).
```

File: doc/articles/getting-started/requirements.md [hunk 2]
```diff
@@ -9,16 +9,20 @@ The platform requirements for the Uno Platform are as follows:
...
 
 ## Visual Studio
 
-- Visual Studio 2019 recommended (Visual Studio 2017 version 15.8 minimum)
+Visual Studio 2019 is recommended (Visual Studio 2017 version 15.8 minimum) for Windows or Mac with the following components installed:
+
+- Universal Windows Platform (if using Visual Studio for Windows)
+- Xamarin (for Android and iOS development)
+- ASP.NET/web component, along with .NET Core 2.2 (for WASM development)
```
Comment (jeromelaban) [inline doc/articles/getting-started/requirements.md]:
```suggestion
- ASP.NET/web component, along with .NET Core 2.2 (for WebAssembly development)
```

File: doc/articles/getting-started/requirements.md (hunk 1 above)
Comment (carldebilly) [reply doc/articles/getting-started/requirements.md]: Works on Opera too ;-)

**Pull request merged**